File: be/SpringBoot/src/main/java/com/common/dto/response/ProfileDTO.java
package com.common.dto.response;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

import java.time.LocalDateTime;

/**
 * DTO đại diện cho dữ liệu Hồ sơ người dùng.
 *
 * @created_at 2026-07-31
 * @author txhoan
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class ProfileDTO {

    Long id;

    String firstName;

    String lastName;

    String phoneNumber;

    String avatarUrl;

    String position;

    String department;

    Long userId;

    LocalDateTime createdAt;

    LocalDateTime modifiedAt;
}

File: be/.Net/Swe.BL/DTO/Response/ProfileDTO.cs
using System;

namespace Swe.BL.DTO.Response;

/// <summary>
/// DTO đại diện cho dữ liệu Hồ sơ người dùng.
/// </summary>
/// <created_at>2026-08-03</created_at>
/// <author>txhoan</author>
public class ProfileDTO
{
    public long Id { get; set; }
    public string? FirstName { get; set; }
    public string? LastName { get; set; }
    public string? PhoneNumber { get; set; }
    public string? AvatarUrl { get; set; }
    public string? Position { get; set; }
    public string? Department { get; set; }
    public long UserId { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime? ModifiedAt { get; set; }
}
